theme/templates/resource-landing-page/resource-header.html

```diff
@@ -27,6 +27,7 @@
                 <span>{{ u|contact }}</span>
             {% endfor %}
         </td></tr>
+        <tr><th>DOI:</th><td>10.4211/hs.{{ cm.short_id }}</td><td><a href="#howToCite">cite</a></td></tr>
```

Comment:
I think it would be good to change the text on the link to "Cite this resource" for better readability.

Reply:
We need to keep it concise. We can run it by Liza @Lizabrazil